src/ops/kernels/cast_kernels.cu

```diff
@@ -0,0 +1,206 @@
+/* Copyright 2021 CMU
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "flexflow/ops/kernels/cast_kernels.h"
+#include "flexflow/utils/cuda_helper.h"
+
+namespace FlexFlow {
+
+CastMeta::CastMeta(FFHandler handle) : OpMeta(handle) {}
+
+namespace Kernels {
+namespace Cast {
+
+/*static*/
+template <typename IDT, typename ODT>
+void forward_kernel_wrapper(const IDT *input_ptr,
+                            ODT *output_ptr,
+                            size_t volume) {
+  cudaStream_t stream;
+  checkCUDA(get_legion_stream(&stream));
+
+  cudaEvent_t t_start, t_end;
+  if (false) {
```

Comment:
Can you create an issue to deal with this when you get a chance?

Reply:
#529

This is what you were talking about right? I feel comfortable getting rid of these blocks in this PR.

Reply:
It should probably be converted to https://github.com/flexflow/FlexFlow/blob/f3f441c3ad731b45204b5f76f3d7aecee85e0288/src/ops/kernels/linear_kernels.cu#L94-L98